lib/pgcli/rails/monkey_patch.rb

```diff
@@ -0,0 +1,19 @@
+module Pgcli
+  module Rails
+    module MonkeyPatch
+      def self.apply!
+        if Rails.gem_version >= Gem::Version.new("7.1.0")
+          require "active_record/connection_adapters/postgresql_adapter"
+          ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.singleton_class.prepend self
+        else
+          ::Rails::DBConsole.prepend self
+        end
+      end
```

Comment:
🗒️ In Rails 7.1, the `find_cmd_and_exec` method moved from `DBConsole` to the `PostgreSQLAdapter` class. The monkey patch for this method is the same; we just need to `prepend` the patch to a different place depending on the Rails version.